src/components/LayoutNaming.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { graphql, useStaticQuery } from "gatsby";
+import { Box, Flex, Grid } from "theme-ui";
+import { Link } from "gatsby";
+
+import { namingEx } from "../../config";
+import Layout from "./Layout";
+import NamingItem from "./NamingItem";
+
+const frame = "25%3A227";
+
+export default (props) => {
+  const images = useStaticQuery(graphql`
+    query NamingItemImageQuery {
+      allFigmaImage {
+        nodes {
```

Comment:
Can we use the `eq` here instead of filtering below?

Reply:
Not sure. How to get access to pageContext data inside the query?

Reply:
Okay me neither so let’s keep and improve later